ui/selectors/selectors.js

```diff
@@ -16,81 +16,81 @@ import { NameType } from '@metamask/name-controller';
 import { TransactionStatus } from '@metamask/transaction-controller';
 import { addHexPrefix, getEnvironmentType } from '../../app/scripts/lib/util';
 import {
-  TEST_CHAINS,
-  BUYABLE_CHAINS_MAP,
-  MAINNET_DISPLAY_NAME,
-  BSC_DISPLAY_NAME,
-  POLYGON_DISPLAY_NAME,
+  ARBITRUM_DISPLAY_NAME,
   AVALANCHE_DISPLAY_NAME,
+  BASE_DISPLAY_NAME,
+  BSC_DISPLAY_NAME,
+  BUYABLE_CHAINS_MAP,
+  CELO_DISPLAY_NAME,
+  CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP,
   CHAIN_ID_TO_RPC_URL_MAP,
+  CHAIN_ID_TOKEN_IMAGE_MAP,
   CHAIN_IDS,
-  NETWORK_TYPES,
-  NetworkStatus,
-  SEPOLIA_DISPLAY_NAME,
-  GOERLI_DISPLAY_NAME,
+  CRONOS_DISPLAY_NAME,
+  CURRENCY_SYMBOLS,
   ETH_TOKEN_IMAGE_URL,
+  FANTOM_DISPLAY_NAME,
+  GNOSIS_DISPLAY_NAME,
+  GOERLI_DISPLAY_NAME,
   LINEA_GOERLI_DISPLAY_NAME,
-  CURRENCY_SYMBOLS,
-  TEST_NETWORK_TICKER_MAP,
   LINEA_GOERLI_TOKEN_IMAGE_URL,
   LINEA_MAINNET_DISPLAY_NAME,
   LINEA_MAINNET_TOKEN_IMAGE_URL,
-  CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP,
-  ARBITRUM_DISPLAY_NAME,
-  OPTIMISM_DISPLAY_NAME,
-  BASE_DISPLAY_NAME,
-  ZK_SYNC_ERA_DISPLAY_NAME,
-  CHAIN_ID_TOKEN_IMAGE_MAP,
-  LINEA_SEPOLIA_TOKEN_IMAGE_URL,
   LINEA_SEPOLIA_DISPLAY_NAME,
-  CRONOS_DISPLAY_NAME,
-  CELO_DISPLAY_NAME,
-  GNOSIS_DISPLAY_NAME,
-  FANTOM_DISPLAY_NAME,
-  POLYGON_ZKEVM_DISPLAY_NAME,
+  LINEA_SEPOLIA_TOKEN_IMAGE_URL,
+  MAINNET_DISPLAY_NAME,
   MOONBEAM_DISPLAY_NAME,
   MOONRIVER_DISPLAY_NAME,
+  NETWORK_TYPES,
+  NetworkStatus,
+  OPTIMISM_DISPLAY_NAME,
+  POLYGON_DISPLAY_NAME,
+  POLYGON_ZKEVM_DISPLAY_NAME,
+  SEPOLIA_DISPLAY_NAME,
+  TEST_CHAINS,
+  TEST_NETWORK_TICKER_MAP,
+  ZK_SYNC_ERA_DISPLAY_NAME,
```

Comment:
Since it's sorted, it's hard for me to asses what's changed 🙈 Was it intended? Is there any addition? Is it your IDE's fault? ;)

Reply:
It was my IDE doing automatic linting! 😮‍💨 I fixed it here [4b2ae4b](https://github.com/MetaMask/metamask-extension/pull/24876/commits/4b2ae4b597e91de9850b4f81ced25eed86ffab25)